lesson-02/task-04.js

```diff
@@ -0,0 +1,13 @@
+const formatString = function (sentense) {
```

Comment:
Дану функцію можна спростити:

```suggestion
const formatString = function (sentense) {
  if (sentense.length > 40) {
       return sentense.slice(0, 40) + "...";
  }
  
  return sentense;
}
```